server/src/main/java/org/opensearch/repositories/RepositoriesService.java

```diff
@@ -346,6 +346,7 @@ protected void doRun() {
         });
     }
 
+    // Note: "voting_only" role has not been implemented yet.
```

Comment:
Are you saying this is unused? If so, can we just go ahead and change MASTER_ROLE below to CLUSTER_MANAGER_ROLE?

Reply:
Yeah, these code about "voting_only" role is a leftover during the removal of non-Apache License codes.
Sure, I can change the ROLE name below, but I just thought it's not worth to change 😄, since it's not used.

Reply:
Replaced the `MASTER_ROLE` directly below, in this commit https://github.com/opensearch-project/OpenSearch/pull/2424/commits/b5837b794376ab7b5bd115bee1f011381ca2ffc9